externs/browser/webgl2_compute.js

```diff
@@ -0,0 +1,1608 @@
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @param {number} v2
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform3i = function(
+    program, location, v0, v1, v2) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @param {number} v2
+ * @param {number} v3
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform4i = function(
+    program, location, v0, v1, v2, v3) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform1ui = function(
+    program, location, v0) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform2ui = function(
+    program, location, v0, v1) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @param {number} v2
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform3ui = function(
+    program, location, v0, v1, v2) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @param {number} v2
+ * @param {number} v3
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform4ui = function(
+    program, location, v0, v1, v2, v3) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform1f = function(
+    program, location, v0) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform2f = function(
+    program, location, v0, v1) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @param {number} v2
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform3f = function(
+    program, location, v0, v1, v2) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} v0
+ * @param {number} v1
+ * @param {number} v2
+ * @param {number} v3
+ * @return {undefined}
+ */
+WebGL2ComputeRenderingContext.prototype.programUniform4f = function(
+    program, location, v0, v1, v2, v3) {};
+
+/**
+ * @param {?WebGLProgram} program
+ * @param {?WebGLUniformLocation} location
+ * @param {number} count
+ * @param {!Int32Array|!Array<number>|!Array<boolean>} data
```

Comment:
Do you know where in the spec `Array<boolean>` is defined to be a subtype of `Int32List`? Asking since `Int32List` was defined as `typedef ([AllowShared] Int32Array or sequence<GLint>) Int32List;` in https://www.khronos.org/registry/webgl/specs/latest/1.0/. Is boolean generally permissible to pass as a `GLint`?

I'm not objecting to this line, since it agrees with the existing webgl externs after https://github.com/google/closure-compiler/commit/030192492b5774fb9d87050a68a049853e115800, but I'd like a more concrete reason if you have one.

Reply:
I'm also not sure about this. Just following existing externs with no more concrete reason.